cfe_internal/CFE_cfengine.cf

```diff
@@ -31,6 +31,14 @@ bundle agent cfe_internal_management
 
     am_policy_hub.enterprise::
 
+      "hub" -> { "Mission Portal", "CFEngine Enterprise" }
+        usebundle => cfe_internal_purge_scheduled_reports_older_than_days( $(def.purge_scheduled_reports_older_than_days) ),
+        ifvarclass => "enable_cfe_internal_purge_scheduled_reports_older_than_days",
```

Comment:
Yes, I know what you mean for phased roll-out across nodes.

However in this case I would just remove it as it won't be a problem for hubs and configurability has a tradeoff with complexity.